sources/settings-migrations/v1.40.0/kubelet-device-plugins-cdi-settings/src/main.rs

```diff
@@ -0,0 +1,234 @@
+use migration_helpers::{error, migrate, Migration, MigrationData, Result};
+use snafu::OptionExt;
+
+#[snafu::report]
+fn main() -> Result<()> {
+    migrate(ReplaceDeviceListStrategy(&[
+        "settings.kubelet-device-plugins.nvidia.device-list-strategy",
```

Comment:
Rather than making `settings.kubelet-device-plugins.nvidia.device-list-strategy` a parameter to your migration, can you make it a constant in your file so that we know that's the only value we are changing?